cypress/integration/scriptSpec.js

```diff
@@ -0,0 +1,53 @@
+import config from '../support/config';
```

Comment:
Nearly every line of this file breaches cypress best practices, please completely refactor.

esp. try to avoid consts.

Reply:
Done

Reply:
Got to [this](https://github.com/bbc/simorgh/blob/9e9e0fcb8364ae53f7d1c147e9c6cf250fa368c2/cypress/integration/scriptSpec.js), Managed to clean 1 test up, but I cant work out how to clean up the second because cypress AFAIK doesn't have a `.filter(() => {})` method